components/intro/Intro.vue

```diff
@@ -62,50 +58,28 @@ export default {
 
 <style lang="postcss" scoped>
 .intro {
-    @apply relative font-serif my-0 mx-auto;
-    max-width: 1040px;
-    @media (max-width: 767px) {
-        @apply px-0 py-4;
-    }
+    @apply relative font-serif px-0 py-4 md:py-0 my-0 mx-auto;
 }
 
 .intro-milestones {
-    @apply grid grid-cols-3;
-    @media (max-width: 767px) {
-        @apply flex flex-col;
-    }
+    @apply flex flex-col;
+    @apply md:grid md:grid-cols-3;
 }
 
 .description-wrapper {
-    @apply flex mb-28;
-    @media (max-width: 767px) {
-        @apply mb-0 flex-col;
-    }
+    @apply flex flex-col md:flex-row mb-0 md:mb-28;
 }
 
 .description-wrapper > div {
-    @apply flex-1;
-    @media (max-width: 767px) {
-        @apply mb-24;
-    }
+    @apply flex-1 mb-24 md:mb-0;
 }
 
 .description-wrapper > div > * {
-    @media (max-width: 767px) {
-        @apply pb-7;
-    }
+    @apply pb-7 md:pb-0;
 }
 
 .info-section-description {
-    @apply pl-14;
-    @media (max-width: 767px) {
-        @apply pl-0 my-0 mx-auto;
-    }
-}
-
-.intro-milestones-title {
```

Comment:
the title should emerge both on web view and mobile view (there is a small error on figma).